CHANGES.rst

```diff
@@ -3,15 +3,14 @@
 Version 8.1.0
 -------------
 
-Unreleased
-
+-   An an argument to ``Path`` which mirrors writable/readable, but for
+    executable bits. :issue:`1961`
```

Comment:
Keep two lines before subsequent headers, this format is used throughout the docs.